### ISSUE TITLE
Recognizes waypoints on initialize but not on scroll

### ISSUE THREAD

**Issue by xd0** [imakewebthings/waypoints#501]
Hi,

Thanks for the help in my earlier issue (https://github.com/imakewebthings/waypoints/issues/493), I didn't have the time to post in that issue before it was closed. I was wondering one more thing, could you take a look at my jsfiddle once again (new version):
https://jsfiddle.net/zlitrox/dggc9xxg/2/

I started basic to get Waypoints working and it works after you did your magic :) I've now added jQuery Tabs and they seem to work until you've navigated on each tab and try to open one of the tabs. Then it ignores first chapter.

Could you please take a look? I'm probably missing something simple here.

**Comment by xd0** [comment]
@imakewebthings Could you please take a look at this aswell? :)

Thanks in advance!

**Comment by imakewebthings** [comment]
@xd0 I don't think the link you posted here is a new version. Am I mistaken?

**Comment by xd0** [comment]
You are correct @imakewebthings! Here's the new one: https://jsfiddle.net/zlitrox/dggc9xxg/8/
